### DISCUSSION TITLE
Using AZD into Jenkins

### DISCUSSION THREAD

**Discussion by julianolarteramos** [Azure/azure-dev#2730] [Ideas]
Has anyone here seen or successfully created an automated deployment pipeline using azd inside Jenkins instead of Github actions or Azure Dev OPS? Do you know of any examples or recommended best practices using azd and jenkins?

**Comment by savannahostrowski** [comment]
Today, we only support azd in GHA or AzDO and don't support running azd in Jenkins. If you want to file a feature request on the repo, we'd be grateful for it!
